src/aac/plugins/gen_json/gen_json_impl.py

```diff
@@ -10,22 +11,51 @@
 )
 from aac.validator import validation
 
+
 plugin_name = "gen-json"
 
 
-def print_json(architecture_files: list[str]) -> PluginExecutionResult:
-    """Print the parsed_models from the parsed architecture_files values in JSON format."""
+def print_json(architecture_files: list[str], output_directory: str = None) -> PluginExecutionResult:
+    """Prints the parsed_models from the parsed architecture_files values in JSON format.
+
+       (optional) if output directory is given to the cli, parsed_models with be writen to
+        a new file in user defined directory
+
+    """
+
+    def to_json(architecture_file: str, output_directory: str) -> str:
+        file_name = os.path.basename(architecture_file)
+        file_ext = os.path.splitext(file_name)[1]
 
-    def to_json(architecture_file: str) -> str:
         with validation(parse_file, architecture_file) as result:
-            return f"File: {architecture_file}\n{json.dumps(result.model, indent=2)}\n"
+
+            if output_directory is None:
+                return f"File: {architecture_file}\n{json.dumps(result.model, indent=2)}\n"
```

Comment:
Should we remove the `File: {architecture_file}\n` portion of the output?  I'm imagining a scenario where someone needs the JSON version for use in a pipeline or something and wouldn't want to have to parse out that first line to use the output.

Reply:
Users can leverage the file output option if they absolutely needed a pipeline solution like that right now, but I think the larger question may be whether or not we want to add in some verbosity configuration that addresses this question for more than just this plugin. 

I think we could modify the existing plugin execution and execution result flow with a clearer delineation between the command output and any success/status messages. We could leverage this by exposing a general argument, like `-q, --quiet` that silences any output that isn't the return value of the command.

Reply:
I'd be up for that.

Reply:
Me too...that's a great idea.